www/video.inc

```diff
@@ -66,15 +66,12 @@ function BuildVideoScripts($testPath)
 *
 * @param mixed $videoDir
 */
-function BuildVideoScript($testPath, $videoDir, $endTime = 0, $extend = false, $startOffset = null)
```

Comment:
I'm pretty sure the whole "video scripts" thing is ancient and no longer used but removing it should probably be done separately.  AVI Synth hasn't been used for a long time and videos are rendered directly by the server now on-demand.

Reply:
Reverted all `BuildVideoScript()` changes